include/aws/s3/private/s3_meta_request_impl.h

```diff
@@ -189,6 +189,8 @@ struct aws_s3_meta_request {
     /* Part size to use for uploads and downloads.  Passed down by the creating client. */
     const size_t part_size;
 
```

Comment:
Maybe put a book member somewhere it’s going to not add extra padding?

Reply:
Could you elaborate more on that?  (This can change per meta request if the part size is too small for the maximum number of parts allowed by the service.)

Reply:
Sorry.. but what is a book member?

Reply:
To Henson's point, let's make this the start of a bitfield at the end of the structure.